subsys/bluetooth/controller/ll_sw/nordic/lll/lll_central_iso.c

```diff
@@ -488,14 +488,17 @@ static void isr_tx(void *param)
 	/* Get reference to CIS LLL context */
 	cis_lll = param;
 
+	/* Acquire rx node for reception */
+	node_rx = ull_iso_pdu_rx_alloc_peek(1U);
+	LL_ASSERT(node_rx);
+
 #if defined(CONFIG_BT_CTLR_LE_ENC)
 	/* Get reference to ACL context */
 	const struct lll_conn *conn_lll = ull_conn_lll_get(cis_lll->acl_handle);
 #endif /* CONFIG_BT_CTLR_LE_ENC */
 
-	/* Acquire rx node for reception */
-	node_rx = ull_iso_pdu_rx_alloc_peek(1U);
-	LL_ASSERT(node_rx);
+	/* PHY */
+	radio_phy_set(cis_lll->rx.phy, PHY_FLAGS_S8);
```

Comment:
I don't understand the change to using `PHY_FLAGS_S8` almost everywhere. Is that something that is defined by the core spec?

Reply:
The second parameter is dont care for radio in Rx state. By default radio h/w by BT Spec. will start receiving in S8 Coded PHY when Coded PHY is the Rx PHY, hence using this define instead of magic value.